dist/_patterns/01-components/01-page-elements/hero.twig

```diff
@@ -0,0 +1,10 @@
+<div class="hero">
+  <div class="hero__content" role=”banner”>
```

Comment:
Will a hero always be the header for a page? If not, I don’t think we should have the banner role.